reference/src/glossary.md

```diff
@@ -0,0 +1,55 @@
+## Glossary
+
+#### Interior mutability
+
+*Interior Mutability* means mutating memory where there also exists a live shared reference pointing to the same memory.
```

Comment:
I'm the last person affected by this, but as far as I know "live" as used here is jargon too. Would it make sense to explain it too, link to an explanation, or paraphrase this informally?

Reply:
TBH I used these terms specifically to avoid having to give a precise definition for things we haven't decided yet.^^

I will add a remark along those lines.

Reply:
I didn't mean spelling out when a shared reference counts as being relevant for interior mutability, I just meant that the term "live" here suggests a concept of "liveness" as in program analysis -- a reference is "still around" or "may be used later" or something like that -- that some readers may not know.

Reply:
Oh I see. I'll add a remark along those lines.